code/game/machinery/limbgrower.dm

```diff
@@ -185,7 +185,10 @@
 			use_power(power)
 			flick("limbgrower_fill", src)
 			icon_state = "limbgrower_idleon"
-			selected_category = params["active_tab"]
+			var temp_category = params["active_tab"]
+			if( ! (temp_category in categories) )
+				return FALSE //seriously come on
+			selected_category = temp_category
```

Comment:
```suggestion
			var/temp_category = params["active_tab"]
			if( ! (temp_category in categories) )
				return FALSE //seriously come on
			selected_category = temp_category
```
style sucks and you should feel bad (the ! ()s are optional the var thing is not)